scheduler/src/cook/plugins/pool.clj

```diff
@@ -0,0 +1,29 @@
+(ns cook.plugins.pool
+  (:require [clojure.tools.logging :as log]
+            [cook.config :as config]
+            [cook.plugins.definitions :refer [PoolSelector]]
+            [cook.plugins.util]
+            [mount.core :as mount]))
+
+(defrecord AttributePoolSelector [attribute-name default-pool]
+  PoolSelector
+  (select-pool [this offer]
+    (or (->> offer :attributes (filter #(= attribute-name (:name %))) first :text)
+        default-pool)))
+
+(defn create-default-plugin-object
```

Comment:
Why is this "default" instead of just `create-plugin`?

Reply:
Done!